server/lib/s3backup/__init__.py

```diff
@@ -206,19 +210,25 @@ def getsize(self, tb):
 
 class S3Connector(Connector):
     def __init__(self, config):
-        self.endpoint_url = config.get('pbench-server-backup', 'endpoint_url')
-        self.bucket_name = config.get('pbench-server-backup', 'bucket_name')
-        self.access_key_id = config.get(
-            'pbench-server-backup', 'access_key_id')
-        self.secret_access_key = config.get(
-            'pbench-server-backup', 'secret_access_key')
-        self.s3client = boto3.client('s3',
-                                     aws_access_key_id='{}'.format(
-                                         self.access_key_id),
-                                     aws_secret_access_key='{}'.format(
-                                         self.secret_access_key),
-                                     endpoint_url='{}'.format(self.endpoint_url))
-        self.transfer_config = None
```

Comment:
We no longer need the `transfer_config` attribute?

Reply:
We do not.